index.ts

```diff
@@ -38,6 +42,26 @@ function getConfig(arc: {
   else return {}
 }
 
+const searchApiFile = 'postdeploy-search.js'
+
+async function executeSearchRequests(cwd: string) {
+  const client = await getSearchClient()
```

Comment:
Move this inside your `if` statement so that we don't bother with configuring the client if it's not needed.

Reply:
Done.